modules/roles/README.md

```diff
@@ -2,14 +2,14 @@
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| create\_instance\_profiles | Controls whether to create IAM instance profiles | string | `"false"` | no |
-| create\_policies | Controls whether to create IAM policies; when false, `policy` must be an ARN | string | `"true"` | no |
-| create\_roles | Controls whether to create IAM roles | string | `"true"` | no |
+| create\_instance\_profiles | Controls whether to create IAM instance profiles | bool | `"false"` | no |
+| create\_policies | Controls whether to create IAM policies; when false, `policy` must be an ARN | bool | `"true"` | no |
+| create\_roles | Controls whether to create IAM roles | bool | `"true"` | no |
 | max\_session\_duration | The maximum session duration (in seconds) for each role, value must be between 60 to 43200 seconds | string | `"43200"` | no |
 | roles | Schema list of IAM roles, consisting of name, policy template, and trust template | list | `<list>` | no |
```

Comment:
May also well also update this description, similar to the `users` update

Reply:
Fixed